### PR TITLE
chore(frontend): extract submit dapp btn into own component

### PR DESCRIPTION
**Pull request by loki344**
# Motivation

Extract component for submitting a dapp to be displayed in the dapp-explorer.

### CODE REVIEW

File: src/frontend/src/lib/components/dapps/SubmitDappButton.svelte [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<script>
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/dapps/SubmitDappButton.svelte:1]: `script lang="ts"`

**Review by peterpeterparker — COMMENTED** [review]
It's me of you forgot to use `SubmitDappButton` in `DappsExplorerSignedIn`?

**Comment by loki344** [comment]
> It's me of you forgot to use `SubmitDappButton` in `DappsExplorerSignedIn`?

wups, yep, added it

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Pull request merged**